### PR TITLE
Add wrangler.toml and fix index.js code

### LINKED ISSUE #16: Add wrangler.toml with type
In anticipation of https://github.com/cloudflare/wrangler/issues/315

### PR DESCRIPTION
**Pull request by victoriabernard92**
fixes #16

This is the default `wrangler generate` template so all nits welcome 😸

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-
 /**
- * Fetch and log a request
+ * Respond with hello worker text
  * @param {Request} request
  */
 async function handleRequest(request) {
-  return new Response('Hello worker!', { status: 200 })
+  return new Response('Hello worker!', {
+    headers: { 'content-type': 'text/plain' },
```
Comment (victoriabernard92) [inline index.js]: status 200 is the default, so I found that irrelevant. I wanted to include one example of modifying `ResponseInit` though and it's best praactice to include `Content-type` anyways so I added that

File: wrangler.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name = "helloworld"
+type = "javascript"
```
Comment (victoriabernard92) [inline wrangler.toml]: Should we make default webpack instead?

Comment (EverlastingBugstopper) [reply wrangler.toml]: Probably. It'd also be great to move `index.js` from the top level and create a `src` directory and point `package.json`:`main` to `./src/index.js`

Comment (victoriabernard92) [reply wrangler.toml]: Ill make an issue for that

Comment (victoriabernard92) [reply wrangler.toml]: https://github.com/cloudflare/worker-template/issues/26

**Pull request merged**